workers/ohsome_quality_analyst/indicators/mapping_saturation/sigmoid_curves.py

```diff
@@ -0,0 +1,59 @@
+"""Sigmoid functions with up to four jumps."""
+
+import numpy as np
+
+
+# Flake8 E741: Ambiguous variable name 'l'
+def sigmoid(x: float, x_0: float, k: float, l: float):  # noqa E741
```

Comment:
Yes I have seen this suggestion in PEP8.
But when `L` is used as single capital variable name two more violations will arise: 
- N803: function name should be lowercase (function names)
- N806: variable in function should be lowercase

That was how I first implemented it. I would like to capitialize the variable name. Then above erros should be added to Flake8 and SonarCloud configuration to be ignored

Reply:
True, both are not ideal. But since the original variable name is a capitalized `L` and you have to ignore some rules for both cases. I would say the “don't use a lowercase `l` because you can mix it up with other letters” argument is the stronger one here.

Reply:
Capital `L` is now used instead of `l`